recipes/stopt/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "stopt" %}
+{% set version = "5.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://gitlab.com/stochastic-control/StOpt
+  git_rev: "v{{ version }}" # (Defaults to "HEAD")
+  # git_depth: 1 # (Defaults to -1/not shallow)
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake >=3.15
+    - make
+    - pkgconfig  # [unix]
+  host:
+    - boost  # TODO: revise to libboost-devel in the future
```

Comment:
You should do this now or otherwise you will pull in a Python dependency.

Reply:
You mean leave boost or update to libboost-devel?

Reply:
I've kept as is and removed the comment so far; let me know if you have different suggestions